Include/ceval.h

```diff
@@ -93,7 +93,12 @@ PyAPI_FUNC(int) Py_GetRecursionLimit(void);
       PyThreadState_GET()->overflowed = 0;  \
     } while(0)
 PyAPI_FUNC(int) _Py_CheckRecursiveCall(const char *where);
-PyAPI_DATA(int) _Py_CheckRecursionLimit;
```

Comment:
This is in the stable API, no?

Reply:
:(  I'm going to revert the _Py_CheckRecursionLimit part of the change for now and circle back after this patch lands.